src/crashlog.cpp

```diff
@@ -177,16 +177,25 @@ void CrashLog::LogConfiguration(std::back_insert_iterator<std::string> &output_i
 			VideoDriver::GetInstance() == nullptr ? "none" : VideoDriver::GetInstance()->GetInfoString()
 	);
 
+	auto log_font = [&](FontSize fs) -> std::string {
```

Comment:
Can't this be a regular function?

Reply:
Like `static FontCache::GetName(FontSize fs)` maybe